config/locales/en.yml

```diff
@@ -189,6 +189,8 @@ en:
         reason_permanently_decanted: "Answer must be internal transfer as you already told us the tenant was permanently decanted from another property owned by this landlord"
         la_general_needs:
           internal_transfer: "Answer cannot be internal transfer as you already told us it's the same landlord on the tenancy agreement and the household had either a fixed-term or lifetime local authority general needs tenancy immediately before this letting"
+        prp:
+          local_housing_referral: "Answer cannot be Nominated by a local housing authority as you already told us it's another landlord on the tenancy agreement"
```

Comment:
Think @Dan13L3 will probably want:
```
"Answer cannot be 'Nominated by a local housing authority' as you already told us it's another landlord on the tenancy agreement"
```

Reply:
@baarkerlounger Yes, please - and to be extra picky, we should keep it lower case to be consistent: 'nominated by a local housing authority'